entfork_sync.go

```diff
@@ -14,8 +14,9 @@ import (
 	"text/template"
 
 	"github.com/google/go-github/v28/github"
-
 	"github.com/sirupsen/logrus"
+
+	tgithub "github.com/mendersoftware/integration-test-runner/client/github"
```

Comment:
because "github" is the import of the GitHub library, I need to use an alias because it is clashing with my github client, which I introduced to be able to mock it in unit tests.

Reply:
Yes, but `t` for `temp`? :thinking:

Reply:
t for `test-runner`, that was my intention :)

Reply:
let's rename it to client_github